cli/cmd/plugin/unmanaged-cluster/test/e2e/unmanaged_cluster_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/test/e2e/utils"
+	"log"
+	"crypto/rand"
+	"math/big"
+	"regexp"
+)
+
+var _ = Describe("UnmanagedCluster", func() {
+	Describe("Unmanaged Cluster testing .ß...", func() {
+
+		// Random Unmanged Cluster Name
+		rand, _ := rand.Int(rand.Reader, big.NewInt(1000))
+		clusterName := "uc" + rand.String() + "test"
+		It("Unmanaged Cluster creation", func() {
+			_, err := e2e.Tanzu(nil, "unmanaged-cluster", "create", clusterName)
+			if err != nil {
+				log.Println("error while unmanaged cluster Creation")
+			}
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Check for package repository", func() {
+			output, err := e2e.Tanzu(nil, "package", "repository", "list", "-A")
+			if err != nil || output == "" {
+				log.Println("error while checking for package repositories")
+			}
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Would be great to also validate that the two package repos are present.

Reply:
Sure @stmcginnis will check for that too. Thanks!